lib/ToggleButton/ToggleButtonAccessibilitySupport.js

```diff
@@ -0,0 +1,28 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name ToggleButtonAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: 'value'}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (was, is, prop) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('aria-pressed', enabled ? String(this.value) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
There's a similar, but more complicated, question to CheckboxItem. In particular, now that you've added `aria-hidden` support, the text within the inner MarqueeText won't be read (at least by ChromeVox) when you set `aria-labelledby` to this.id. However, you can't blindly set it to `this.$.client.getId()` as with CheckboxItem because client may not exist if the ToggleButton was populated with other components.

To support the appropriate extensibility, I'd further suggest that you add a property, perhaps `accessibilityLabeledBy` to this mixin that allows consumers or extenders of ToggleButton specify the name of the control that contains the label. It could default to 'client' so the common scenario of a content-only ToggleButton would work as is.